src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -145,6 +145,13 @@ public function getStartTime();
      */
     public function getCacheDir();
 
+    /**
+     * Gets the temporary directory.
+     *
+     * @return string The temporary directory
+     */
+    public function getTmpDir();
```

Comment:
I suspect this is a BC break, I'm not sure what the policy is on master.

If we want to remove that, we can probably check using `method_exists()` before calling it, or any other method you are used to do in those cases?

Reply:
https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces
`Add method: No`

Reply:
You can add it with PHPDoc on the interface: `@method string getTmpDir() Returns the temporary directory - not implementing it is deprecated since Symfony 5.1` and then you method_exists() and fall back on getCacheDir() when it's not implemented.